.env.production

```diff
@@ -1 +1,4 @@
 NEXT_PUBLIC_RUNTIME_ENV=production
+# DSN of pingcap/tug-website
+SENTRY_DSN=https://cc3bce60904940f8a8e5795ef6be2ae7@o226447.ingest.sentry.io/5756513
+NEXT_PUBLIC_SENTRY_DSN=$SENTRY_DSN
```

Comment:
Can you double-check if it applies a `NEXT_PUBLIC` prefix, the env variable would be exposed to the node environment as well?

Reply:
Node (server) uses `SENTRY_DSN`.